Loop Status Extension/StatusViewController.swift

```diff
@@ -205,11 +215,20 @@ class StatusViewController: UIViewController, NCWidgetProviding {
                 if let eventualGlucose = predictedGlucose.last {
                     let formatter = NumberFormatter.glucoseFormatter(for: eventualGlucose.unit)
 
+                    let activeInsulinString: String
+                    if let activeInsulin = context.activeInsulin, let valueStr = insulinFormatter.string(from:NSNumber(value:activeInsulin))
+                    {
+                        activeInsulinString = valueStr
+                    } else {
+                        activeInsulinString = "--"
+                    }
+
                     if let eventualGlucoseNumberString = formatter.string(from: NSNumber(value: eventualGlucose.quantity.doubleValue(for: unit))) {
                         subtitleLabel.text = String(
                             format: NSLocalizedString(
-                                "Eventually %1$@ %2$@",
-                                comment: "The subtitle format describing eventual glucose. (1: localized glucose value description) (2: localized glucose units description)"),
+                                "%1$@ U IOB;   BG eventually %2$@ %3$@",
```

Comment:
UI shouldn't be positioned using a semicolon and two space characters. The IOB value belongs in its own label. Perhaps in the lower left

Reply:
OK, agreed about the separate label.  I thought of that at the start, but wimped out because it was easier to add it to the existing label.  But it's good motivation for me to keep learning more Swift. 

But if it's to remain visible in the collapsed view of the widget (see screenshot), I think it makes sense to keep it more or less where it is.   And I also think that from a usability standpoint it's useful to have both numbers relatively close to each other, to take them in in one glance. 

Perhaps both labels could be on that line, with one flush left and one flush right?  Are there potential issues with them overlapping? 

![iob_widget](https://user-images.githubusercontent.com/2545845/33799995-66d01cc2-dd05-11e7-8fe2-2fd7b061db4c.png)

Reply:
thanks for posting the screenshot. I agree that it's a little weird to put the two strings together. It's also a little weird that one of them is "<##> IOB" and the other is "BG <##>". I like the flush left, flush right option. So combining the two and normalizing the strings would yield:

IOB: 1.1U        BG: Eventually 66 mg/dL
|<-------        ---------------------->|

I don't think you'll have an issue with them overlapping.